src/main/java/com/example/icebutler_server/global/exception/ExceptionAdvice.java

```diff
@@ -25,4 +31,26 @@ protected ResponseCustom handleMethodArgumentNotValidException(MethodArgumentNot
         ReturnCode returnCode = ReturnCode.findByCode(fieldError.getDefaultMessage());
         return ResponseCustom.error(returnCode);
     }
+
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    @ExceptionHandler(
+            value = {
+                    HttpMessageNotReadableException.class,
+                    HttpRequestMethodNotSupportedException.class,
+                    MissingServletRequestParameterException.class,
+                    MethodArgumentTypeMismatchException.class,
+                    DateTimeParseException.class
+            }
+    )
+    protected ResponseCustom handleBaseException(Exception e) {
+        return ResponseCustom.error(ReturnCode.INVALID_PARAM);
+    }
```

Comment:
```suggestion
    @ResponseStatus(HttpStatus.BAD_REQUEST)
    @ExceptionHandler(
            value = {
                    HttpMessageNotReadableException.class,
                    HttpRequestMethodNotSupportedException.class,
                    MissingServletRequestParameterException.class,
                    MethodArgumentTypeMismatchException.class,
                    DateTimeParseException.class
            }
    )
    protected ResponseCustom handleBaseException(Exception e) {
    // 제안 부분
        FieldError error = Objects.requireNonNull(e.getFieldError());
        return ResponseCustom.error(ReturnCode.INVALID_PARAM, String.format("%s", error.getDefaultMessage()));
    }
```

INVALID_PARAM 으로 공통으로 처리하는 방법 간편하고 좋은 것 같아요!
여기서 추가로 `error.getDefaultMessage()` 통해서 어떤 컬럼에서 문제가 발생했는지 알 수 있는데,
프론트가 필요하면 ResponseCustom의 data로 같이 넘겨주고, 필요 없다면 내부 로그로 저장시켜두는거 어떤가요 ?
(테스트나 문제 발생 대비)

Reply:
이건 또 무슨 새로운 기능인거죠?! 근데 getDefaultMessage를 하면 각 에러별로 내장 에러메세지가 나오는데 그걸 그대로 응답에 보여주는 건 좀 그럴 것 같아요! 내부 로그에만 기록하는 건 어떤가요??

Reply:
로그 기록하는 거 좋아요! 로그는 관련해서 같이 따로 얘기해서 적용하는걸로 ~